kafka/Dockerfile

```diff
@@ -16,6 +16,17 @@ ARG opensearch=opensearch-connector-for-apache-kafka-3.1.0.tar
 ADD --chown=kafka --chmod=600 https://github.com/aiven/opensearch-connector-for-apache-kafka/releases/download/v3.1.0/${opensearch} \
     /kafka/connect/
 
+ARG iamauth=v2.2.0
+
+ADD --chown=kafka --chmod=600 https://github.com/aws/aws-msk-iam-auth/releases/download/${iamauth}/aws-msk-iam-auth-${iamauth}-all.jar \
```

Comment:
@NovaLogicDev 
The file name does not contain the letter 'v' in the version segment. The correct link is: https://github.com/aws/aws-msk-iam-auth/releases/download/v2.2.0/aws-msk-iam-auth-2.2.0-all.jar.

I tested locally with:

```
ARG iamauth=**2.2.0**
ADD --chown=kafka --chmod=600 https://github.com/aws/aws-msk-iam-auth/releases/download/**v${iamauth}**/aws-msk-iam-auth-${iamauth}-all.jar \ 
```
Working as expected. Thanks